src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,15 @@
+@mixin hand($var, $name, $time, $angle, $step) {
+  position: absolute;
+  transform-origin: bottom center;
+  animation: $name $time steps($step, end) forwards;
```

Comment:
Add infinite loop to the animation to make the stopwatch hands run continuously.

Reply:
Not clear for me. Shoul I set animation-iteration-count to infinite?

Reply:
Add the `infinite` keyword to this string to make a loop of your animation.